docker-compose.yaml

```diff
@@ -0,0 +1,10 @@
+version: "3"
+
+services: 
+  auth:
+    container_name: "jwt-auth"
+    build: 
+      context: "./"
+      dockerfile: "./docker/auth.Dockerfile"
+    ports: 
```

Comment:
We need actual env vars in compose not ones from default inside Dockerfile.

Reply:
Update with new commit